src/v2/styles/Message/Message-layout.scss

```diff
@@ -473,3 +489,20 @@
     transform: none;
   }
 }
+
+.str-chat__message-edited-timestamp {
+  overflow: hidden;
+  transition: height 0.1s;
+
+  &--open {
+    height: var(--str-chat__message-edited-timestamp-height, 1rem);
+  }
+
+  &--collapsed {
+    height: 0;
+  }
+}
+
+.str-chat__message-text--pointer-cursor {
```

Comment:
@myandrienko I've added this here -> SDK's can apply this dynamically

Reply:
Angular logic for reference: https://github.com/GetStream/stream-chat-angular/pull/563/files#diff-1a38d0771c6ea73cc1bfd32b5699ffdc969cbcd65f0f99fef492517b2f201ed1R179-R183 - I know, I know that expression is a bit too complicated, but it handles the following cases:
- unsent message
- bounced message
- edited message